lib/runtime/wasmer/imports.go

```diff
@@ -811,17 +811,17 @@ func ext_trie_blake2_256_root_version_1(context unsafe.Pointer, dataSpan C.int64
 	// allocate memory for value and copy value to memory
 	ptr, err := runtimeCtx.Allocator.Allocate(32)
 	if err != nil {
-		logger.Errorf("[ext_trie_blake2_256_root_version_1]: %s", err)
+		logger.Errorf("failed allocating: %s", err)
 		return 0
 	}
 
 	hash, err := t.Hash()
 	if err != nil {
-		logger.Errorf("[ext_trie_blake2_256_root_version_1]: %s", err)
+		logger.Errorf("failed computing trie Merkle root hash: %s", err)
```

Comment:
```suggestion
		logger.Errorf("failed computing trie merkle root hash: %s", err)
```

Reply:
very niche nit, but Merkle is a proper noun so a capital letter looks appropriate 😄